.gitignore

```diff
@@ -8,6 +8,9 @@ yarn-error.log*
 lerna-debug.log*
 .pnpm-debug.log*
 
+#misc
+.DS_Store
```

Comment:
Tu peux supprimer les fichiers DS_store qui ont été rajouté stp?
![image](https://github.com/user-attachments/assets/4da1c614-f782-43b2-adee-8a0b2547b097)

Command you can use : `git rm --cached <file>`

Reply:
pour le git ignore tu vas aussi avoir besoin de mettre `*.DS_Store` so it can catch DS files in sub folders